apps/web/src/components/Home/VisitAlternate.tsx

```diff
@@ -6,7 +6,7 @@ import { Icon } from '../Icon'
 
 const VisitAlternate = () => {
   const { isMobile } = useLayoutStore()
-  const isTestnet = process.env.NEXT_PUBLIC_CHAIN_ID === '5' ? true : false
+  const isTestnet = process.env.NEXT_PUBLIC_NETWORK_TYPE === 'testnet' ? true : false
```

Comment:
worth to export as a shareable const? seems like a useful utility

Reply:
added this in and changed most of the NETWORK_TYPE references to it